soroban-sdk/src/lib.rs

```diff
@@ -299,6 +299,45 @@ pub use soroban_sdk_macros::contractimport;
 /// ```
 pub use soroban_sdk_macros::contractimpl;
 
+/// Adds a serialized SCMetaEntry::SCMetaV0 to the WASM contracts custom section
+/// under the section name 'contractmetav0'. Contract developers can use to to
+/// append metadata to their contract.
+///
+/// ### Examples
+///
+/// ```
+/// use soroban_sdk::{contractimpl, contractmeta, vec, BytesN, Env, Symbol, Vec};
+///
+/// pub struct HelloContract;
+///
+/// #[contractimpl]
+/// impl HelloContract {
+///     pub fn hello(env: Env, to: Symbol) -> Vec<Symbol> {
+///         vec![&env, Symbol::short("Hello"), to]
+///     }
+/// }
+///
+/// contractmeta!(key="desc", val="hello world contract");
```

Comment:
Could we move the use of the macro to the top of the example above the pub struct, since it is the primary thing of interest?

Reply:
Done